src/ansys/fluent/core/services/field_data.py

```diff
@@ -434,13 +434,13 @@ def __init__(
     @deprecate_argument(
         old_arg="surface_names",
         new_arg="surfaces",
-        converter=lambda old_arg_val: old_arg_val,
+        converter=lambda old_arg_val: old_arg_val if old_arg_val else None,
```

Comment:
This will pass `None` to the new argument but `None` is not allowed from its typehint. The correct lambda is `lambda old_arg_val: old_arg_val or []`.

Also, `PyFluentDeprecationWarning` is the default value for `warning_cls`.

Reply:
Done, Thank you.